### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by hao-nan-li**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 ## 4.81.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_biglake_catalog` ([#6152](https://github.com/hashicorp/terraform-provider-google-beta/pull/6152))
+* **New Resource:** `google_redis_cluster` ([#6158](https://github.com/hashicorp/terraform-provider-google-beta/pull/6158))
+* **New Resource:** `google_biglake_database` ([#6161](https://github.com/hashicorp/terraform-provider-google-beta/pull/6161))
+* **New Resource:** `google_compute_network_attachment` ([#6159](https://github.com/hashicorp/terraform-provider-google-beta/pull/6159))
+
+IMPROVEMENTS:
+* compute: made the field `distribution_policy_target_shape` of `google_compute_region_instance_group_manager` not cause recreation of the resource. ([#6156](https://github.com/hashicorp/terraform-provider-google-beta/pull/6156))
```
Comment (ScottSuarez) [inline CHANGELOG.md:10]:
```suggestion
* compute: allowed the field `distribution_policy_target_shape` of `google_compute_region_instance_group_manager` to be updatable. ([#6156](https://github.com/hashicorp/terraform-provider-google-beta/pull/6156))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+* compute: promoted the `ssl_policy` field on the `google_compute_region_target_https_proxy` resource to GA. ([#6138](https://github.com/hashicorp/terraform-provider-google-beta/pull/6138))
+* container: added `enable_fqdn_network_policy` field to `google_container_cluster` ([#6157](https://github.com/hashicorp/terraform-provider-google-beta/pull/6157))
+* container: added `node_config.confidential_compute` field to `google_container_node_pool` resource ([#6166](https://github.com/hashicorp/terraform-provider-google-beta/pull/6166))
+* datastream: made `password` mutable in `google_datastream_connection_profile` not cause recreation of the resource. ([#6140](https://github.com/hashicorp/terraform-provider-google-beta/pull/6140))
```
Comment (ScottSuarez) [inline CHANGELOG.md:14]:
```suggestion
* datastream: allowed `password` of `google_datastream_connection_profile` to be updatable. ([#6140](https://github.com/hashicorp/terraform-provider-google-beta/pull/6140))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
 
+FEATURES:
+* **New Resource:** `google_biglake_catalog` ([#6152](https://github.com/hashicorp/terraform-provider-google-beta/pull/6152))
+* **New Resource:** `google_redis_cluster` ([#6158](https://github.com/hashicorp/terraform-provider-google-beta/pull/6158))
+* **New Resource:** `google_biglake_database` ([#6161](https://github.com/hashicorp/terraform-provider-google-beta/pull/6161))
+* **New Resource:** `google_compute_network_attachment` ([#6159](https://github.com/hashicorp/terraform-provider-google-beta/pull/6159))
+
+IMPROVEMENTS:
+* compute: made the field `distribution_policy_target_shape` of `google_compute_region_instance_group_manager` not cause recreation of the resource. ([#6156](https://github.com/hashicorp/terraform-provider-google-beta/pull/6156))
+* compute: promoted the `ssl_policy` field on the `google_compute_region_target_https_proxy` resource to GA. ([#6138](https://github.com/hashicorp/terraform-provider-google-beta/pull/6138))
```
Comment (ScottSuarez) [inline CHANGELOG.md:11]:
```suggestion
```
GA only

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+* container: added `node_config.confidential_compute` field to `google_container_node_pool` resource ([#6166](https://github.com/hashicorp/terraform-provider-google-beta/pull/6166))
+* datastream: made `password` mutable in `google_datastream_connection_profile` not cause recreation of the resource. ([#6140](https://github.com/hashicorp/terraform-provider-google-beta/pull/6140))
+* dialogflowcx: added `response_type`, `channel`, `payload`, `conversation_success`, `output_audio_text`, `live_agent_handoff`, `play_audo`, `telephony_transfer_call`, `reprompt_event_handlers`, `set_parameter_actions`, and `conditional_cases` fields to `google_dialogflow_cx_page` resource ([#6168](https://github.com/hashicorp/terraform-provider-google-beta/pull/6168))
+* dialogflowcx: added `response_type`, `channel`, `payload`, `conversation_success`, `output_audio_text`, `live_agent_handoff`, `play_audo`, `telephony_transfer_call`, `set_parameter_actions`, and `conditional_cases` fields to `google_dialogflow_cx_flow` resource ([#6168](https://github.com/hashicorp/terraform-provider-google-beta/pull/6168))
```
Comment (ScottSuarez) [inline CHANGELOG.md:16]:
```suggestion
* dialogflowcx: added `response_type`, `channel`, `payload`, `conversation_success`, `output_audio_text`, `live_agent_handoff`, `play_audo`, `telephony_transfer_call`, `set_parameter_actions`, and `conditional_cases` fields to `google_dialogflow_cx_flow` resource ([#6168](https://github.com/hashicorp/terraform-provider-google-beta/pull/6168))
```

duplicate notes

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+* gkehub: added `Namespace`, `Scope`, `MembershipBinding`, `MembershipRBACRoleBinding`, `ScopeRBACRoleBinding` resources ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
```
Comment (ScottSuarez) [inline CHANGELOG.md:17]:
```suggestion
```

these are new resource and should be added to the new resource section

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
 ## 4.81.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_biglake_catalog` ([#6152](https://github.com/hashicorp/terraform-provider-google-beta/pull/6152))
+* **New Resource:** `google_redis_cluster` ([#6158](https://github.com/hashicorp/terraform-provider-google-beta/pull/6158))
+* **New Resource:** `google_biglake_database` ([#6161](https://github.com/hashicorp/terraform-provider-google-beta/pull/6161))
+* **New Resource:** `google_compute_network_attachment` ([#6159](https://github.com/hashicorp/terraform-provider-google-beta/pull/6159))
```
Comment (ScottSuarez) [inline CHANGELOG.md:7]:
```suggestion
* **New Resource:** `google_compute_network_attachment` ([#6159](https://github.com/hashicorp/terraform-provider-google-beta/pull/6159))
* **New Resource:** `google_gke_hub_membership_binding` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_namespace` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_scope` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_scope_iam_member` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_scope_iam_policy` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_membership_binding` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
* **New Resource:** `google_gke_hub_scope_rbac_role_binding` ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
```

Double check these before committing

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+* dialogflowcx: added `response_type`, `channel`, `payload`, `conversation_success`, `output_audio_text`, `live_agent_handoff`, `play_audo`, `telephony_transfer_call`, `set_parameter_actions`, and `conditional_cases` fields to `google_dialogflow_cx_flow` resource ([#6168](https://github.com/hashicorp/terraform-provider-google-beta/pull/6168))
+* gkehub: added `Namespace`, `Scope`, `MembershipBinding`, `MembershipRBACRoleBinding`, `ScopeRBACRoleBinding` resources ([#6170](https://github.com/hashicorp/terraform-provider-google-beta/pull/6170))
+* gkehub: promoted the `google_gke_hub_feature_membership` resource to GA ([#6136](https://github.com/hashicorp/terraform-provider-google-beta/pull/6136))
+* iam: added `web_sso_config.additional_scopes` field to `google_iam_workforce_pool_provider` resource under ([#6145](https://github.com/hashicorp/terraform-provider-google-beta/pull/6145))
```
Comment (ScottSuarez) [inline CHANGELOG.md:19]:
```suggestion
* iam: added `web_sso_config.additional_scopes` field to `google_iam_workforce_pool_provider` resource. ([#6145](https://github.com/hashicorp/terraform-provider-google-beta/pull/6145))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
...
+* iamworkforcepool: added `jwksJson` field to `WorkforcePoolProvider` resource (beta) ([#6153](https://github.com/hashicorp/terraform-provider-google-beta/pull/6153))
```
Comment (ScottSuarez) [inline CHANGELOG.md:20]:
```suggestion
* iamworkforcepool: added `jwksJson` field to `WorkforcePoolProvider` resource ([#6153](https://github.com/hashicorp/terraform-provider-google-beta/pull/6153))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,33 @@
...
+* monitoring: added `synthetic_monitor` to `google_monitoring_uptime_check_config` resource ([#6148](https://github.com/hashicorp/terraform-provider-google-beta/pull/6148))
+
+BUG FIXES:
+* cloudrunv2: changed `template.volumes.secret.items.mode` field in `google_cloud_run_v2_job` resource to a non-required field. ([#6154](https://github.com/hashicorp/terraform-provider-google-beta/pull/6154))
+* cloudrunv2: changed `template.volumes.secret.items.mode` field in `google_cloud_run_v2_service` resource to a non-required field. ([#6154](https://github.com/hashicorp/terraform-provider-google-beta/pull/6154))
+* filestore: fixed a bug causing permadiff on `reserved_ip_range` field in `google_filestore_instance` ([#6143](https://github.com/hashicorp/terraform-provider-google-beta/pull/6143))
+* identityplatform: fixed a permadiff on `authorized_domains` in `google_identity_platform_config` resource ([#6137](https://github.com/hashicorp/terraform-provider-google-beta/pull/6137))
+* provider: improved error message when resource creation fails to to invalid API response ([#6149](https://github.com/hashicorp/terraform-provider-google-beta/pull/6149))
```
Comment (ScottSuarez) [inline CHANGELOG.md:28]: Perhaps this should be a note instead of a bug fix?

**Review by ScottSuarez — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hao-nan-li) [reply CHANGELOG.md:18]: It was suggested on the TPG release note to make this note focusing on the user impact. Thanks for the suggestion here.

File: CHANGELOG.md (hunk 4 above)
Comment (hao-nan-li) [reply CHANGELOG.md:23]: Thanks for the message, these are nearly identical except the fields are added to 2 different resources: 
google_dialogflow_cx_page and google_dialogflow_cx_flow

Comment (ScottSuarez) [reply CHANGELOG.md:22]: Thanks for checking this!

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**